mitodl/hugo-course-publisher
# add topic facet to search

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #215

#### What's this PR do?

this pulls the course-search-utils package in and uses the functionality it affors to implement a 'topic' facet.

#### How should this be manually tested?

Try it out and make sure it operates the way you'd expect. It should have more or less the same behavior as in Open.

#### Screenshots (if appropriate)

![Screenshot from 2020-09-22 09-50-57](https://user-images.githubusercontent.com/6207644/93932885-d689ea00-fcee-11ea-98b2-ece770c22d9b.png)
